packages/ibm-products/src/components/DataSpreadsheet/DataSpreadsheet.tsx

```diff
@@ -291,7 +291,11 @@ export let DataSpreadsheet = React.forwardRef(
                   activeCellCoordinates?.column
                 ]
               : null;
-          setActiveCellContent(activeCellFullData.render('Cell'));
+          if (activeCellFullData) {
+            setActiveCellContent(activeCellFullData.render('Cell'));
+          } else {
+            setActiveCellContent(null);
+          }
```

Comment:
I think we can replace this with the Nullish coalescing operator (??) and optional chaining (.?).
`setActiveCellContent(activeCellFullData?.render('Cell') ?? null);`

Reply:
Sure

Reply:
Done, thanks!